common/app/Map/Challenge.jsx

```diff
@@ -29,7 +30,7 @@ function makeMapStateToProps(_, { dashedName }) {
     userSelector,
     challengeMapSelector,
     (
-      { challengeMap: userChallengeMap },
+      { challengeMap: userChallengeMap, currentChallengeId},
```

Comment:
This will only work if I am signed in. 

It would be better to use the current challenge `dashedName` and compare that to `state.app.currentChallenge` or `params.dashedName`

Reply:
thanks i have updated the PR.